package.json

```diff
@@ -7,17 +7,19 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "patch-package": "^6.4.7",
```

Comment:
i did not know about this library. Huge improvement to some of the things I've had to do in the past.

Reply:
I like it, life-saver :)